src/action.ts

```diff
@@ -257,7 +254,7 @@ async function action(pr: PullRequest): Promise<void> {
   );
 
   // Add comment with Testing Farm request/result to Pull Request
-  if (getBooleanInput('create_issue_comment')) {
+  if (getBooleanInput('create_issue_comment') && pr.isInitialized()) {
```

Comment:
I woule counterwise this condition. First check if PR is initialized and then get boolean value.

Reply:
Sure, I'll change the order.